qr-code/node/web/frontend/pages/index.jsx

```diff
@@ -6,8 +6,8 @@ import {
   Page,
   SkeletonBodyText,
 } from "@shopify/polaris";
-import { useAppQuery } from "../hooks";
 import { QRCodeIndex } from "../components";
+import { useAppQuery } from "../hooks";
```

Comment:
looks like this might be missing here: https://shopify-dev.shopify-dev-minimal-72uq.paulo-margarido.us.spin.dev/apps/getting-started/build-app-example/frontend#create-your-home-page-with-an-empty-state

Reply:
This particular line gets added later on: https://shopify-dev.shopify-dev-minimal-72uq.paulo-margarido.us.spin.dev/apps/getting-started/build-app-example/connect#step-4-connect-the-list-view-to-display-saved-qr-codes